manifests/repo/redhat.pp

```diff
@@ -7,18 +7,18 @@
   }
 
   yumrepo { 'Zend':
-    baseurl  => "http://repos.zend.com/zend-server/${zendserver::zend_server_version}/rpm/${::architecture}",
```

Comment:
@ubellavance can anything be passed as the zendserver::zend_server_version, or is it only a version number such as x.y?
The convention on the Zend repo is for the x.y version to symlink to whatever latest version's available in the repo, in that case 7.0 -> 7.0.0_update8. However I can see that the flexibility to pin to a specific version would be helpful.